examples/creating-a-driven-control.ipynb

```diff
@@ -40,17 +46,17 @@
     "\n",
     "Q-CTRL Open Controls can create driven controls according to the following dynamically corrected gate protocols:\n",
     "\n",
-    "1. `primitive`\n",
-    "2. `BB1`\n",
-    "3. `SK1`\n",
-    "4. `CORPSE`\n",
-    "5. `WAMF1`\n",
-    "6. `SCROFULOUS`\n",
-    "7. `COPRSE in BB1`\n",
-    "8. `CORPSE in SK1`\n",
-    "9. `CORPSE in SCROFULOUS`\n",
+    "1. [`new_primitive_control`](https://docs.q-ctrl.com/references/python/qctrl-open-controls/qctrlopencontrols/new_primitive_control.html)\n",
```

Comment:
```suggestion
    "1. [Primitive](https://docs.q-ctrl.com/references/python/qctrl-open-controls/qctrlopencontrols/new_primitive_control.html)\n",
```

IMO it reads a little strange to say "... according to the following dynamically corrected gate protocols: new_primitive_control, new_bb1_control, ...", because "new_primitive_control" isn't a dynamically corrected gate protocol, it's a function. I'd suggest changing the list items like this.

Reply:
Good point. Fixed now.